src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableWriter.class.php

```diff
@@ -354,6 +354,18 @@ public function Delete($sId = null)
         TCMSLogChange::WriteTransaction($aQuery);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function DeleteConnectedRecordReferences()
+    {
+        $this->DeleteIdConnectedRecordReferences();
+        $this->DeleteMltConnectedRecordReferences();
+
+        // delete all related records targeting the deleted table
+        $this->deleteMultiTableRecordReferences($this->oTableConf->sqlData['name']);
```

Comment:
Name the method (for example) `deleteRecordReferencesTargeting($table)` and get rid of the comment?

Reply:
The method needs info about what records are deleted (MultiTable references). The difference of this call is only that i only set the table. This handles the deletion of the whole table where there is no deletion record by record first. So i  introduced a new method called: deleteMultiTableRecordReferencesForDeletedTable

Reply:
And the method can call the parent here i found out.